### PR TITLE
docs: Add integration doc for inactive validators

### PR DESCRIPTION
**Pull request by p-offtermatt**
## Description

Closes: #XXXX

Improve docs for inactive vals, aimed at integrators.

We will link to these as part of an outreach to block explorers, wallets, staking platforms, and other integrators.

Please review the docs locally to see how this looks on the web:

`make build-docs-local`

In particular, interested in opinions on where this page should be (is it ok under `Upgrading`, should there be an extra section called `Integration`; other ideas on where it could be?)

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] included the correct `docs:` prefix in the PR title
- [x] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/interchain-security/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary 
- [x] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] Confirmed the correct `docs:` prefix in the PR title
- [ ] Confirmed all author checklist items have been addressed 
- [ ] Confirmed that this PR only changes documentation
- [ ] Reviewed content for consistency
- [ ] Reviewed content for spelling and grammar
- [ ] Tested instructions (if applicable)
- [ ] Checked that the documentation website can be built and deployed successfully (run `make build-docs`)

## Summary by CodeRabbit

- **New Features**
	- Introduced a detailed integration guide for the inactive validators feature, enhancing participation in Interchain Security.
	- Clarified the distinction between bonded and active validators, including implications for rewards and governance.
	- Updated information on query changes, specifically regarding the bonded_tokens reflecting all bonded validators.
	- Added a new "Integrators Guide" to improve documentation organization and accessibility.

### CODE REVIEW

**Review by stana-miric — APPROVED** [review]

File: docs/docs/upgrading/integrating_inactive_validators.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+*active* validators are all validators that actively participate in consensus on the Cosmos Hub.
+
+Before the introduction of the feature, these two terms were equivalent: every bonded validator was active, and every active validator was bonded. *After* the introduction of this feature, it still holds that every active validator is bonded, but not every bonded validator is active.
+
+Importantly, only *active* validators receive inflation rewards from ATOM; only *active* validators may vote on behalf of their delegators in governance, and *only active* validators can get slashed for downtime (because only those validators participate in consensus and produce blocks). Apart from these differences, *bonded but inactive* validators are just like *active* validators - they can receive delegations, and they can validate on consumer chains (and receive rewards for this) just like active validators.
+
+The following queries will change after this upgrade:
+
+### `/cosmos/staking/v1beta1/pool`
```
Comment (insumity) [inline docs/docs/upgrading/integrating_inactive_validators.md:21]: nit: Might make sense to mention (`query staking pool`) as well here for this query so it's easier to realize the query that changes (?)

File: docs/docs/upgrading/integrating_inactive_validators.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+The *bonded_tokens* will include the stake of all *bonded* validators. As the number of bonded validators will be increased as part of the upgrade, the number of bonded_tokens is expected to have a sudden increase after the upgrade is applied.
```
Comment (insumity) [inline docs/docs/upgrading/integrating_inactive_validators.md:23]: nit: maybe use ` and have `bonded_tokens` here and later when you talk about `bonded_tokens`.

File: docs/docs/upgrading/integrating_inactive_validators.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+### All queries in the staking module that return a `Validator`
+
+All *bonded* validators will show with `Status=Bonded` . To identify *active* validators, query the validator set from Tendermint (https://docs.cometbft.com/v0.37/rpc/#/Info/validators), which will return the set of all *active* validators.
```
Comment (insumity) [inline docs/docs/upgrading/integrating_inactive_validators.md:27]: nit: Also add query here `query comet-validator-set [height]`

**Comment by insumity** [comment]
The right side of the page looks peculiar because it just shows the `/cosmos/staking/v1beta1/pool` and `All queries in the staking module that return a Validator` as sections:
page

Having this under `Upgrading` might be easily missed. I wonder if it would make sense to have a new section called "Integrators Guide" that would contain the always-up-to-date information for block explorers, etc.

File: docs/docs/upgrading/integrating_inactive_validators.md (hunk 1 above)
Comment (p-offtermatt) [reply docs/docs/upgrading/integrating_inactive_validators.md:21]: Hm, yeah it's hard to tell what people use/recognize more. This is the path of the GRPC query

Comment (p-offtermatt) [reply docs/docs/upgrading/integrating_inactive_validators.md:21]: I can just add the other option, too

File: docs/docs/upgrading/integrating_inactive_validators.md (hunk 2 above)
Comment (p-offtermatt) [reply docs/docs/upgrading/integrating_inactive_validators.md:23]: This comment was very hard to understand due to the code block, lol. I think you mean bonded_tokens should be `bonded_tokens`, which I agree with

**Comment by p-offtermatt** [comment]
Thanks for the feedback, I have made an extra section called "Integrators Guide"

File: docs/docs/integrators/integrating_inactive_validators.md [hunk 4]
```diff
@@ -18,10 +18,10 @@ Importantly, only *active* validators receive inflation rewards from ATOM; only
...
 
-The *bonded_tokens* will include the stake of all *bonded* validators. As the number of bonded validators will be increased as part of the upgrade, the number of bonded_tokens is expected to have a sudden increase after the upgrade is applied.
+The `bonded_tokens` will include the stake of all *bonded* validators. As the number of bonded validators will be increased as part of the upgrade, the number of `bonded_tokens` is expected to have a sudden increase after the upgrade is applied.
 
-### All queries in the staking module that return a `Validator`
+* All queries in the staking module that return a `Validator`
 
-All *bonded* validators will show with `Status=Bonded` . To identify *active* validators, query the validator set from Tendermint (https://docs.cometbft.com/v0.37/rpc/#/Info/validators), which will return the set of all *active* validators.
\ No newline at end of file
+All *bonded* validators will show with `Status=Bonded` . To identify *active* validators, query the validator set from Tendermint (https://docs.cometbft.com/v0.37/rpc/#/Info/validators or `query comet-validator-set [height]`), which will return the set of all *active* validators.
```
Comment (insumity) [inline docs/docs/integrators/integrating_inactive_validators.md:27]: nit: `gaiad query staking pool` versus `query comet-validator-set [height]` here. Maybe remove `gaiad` from the previous query or add `gaiad` in this query as well.

**Review by insumity — APPROVED** [review]

File: docs/docs/integrators/integrating_inactive_validators.md (hunk 4 above)
Comment (p-offtermatt) [reply docs/docs/integrators/integrating_inactive_validators.md:27]: Thx, removed gaiad since this is the ICS repo

**Pull request merged**